docs/Migrating.md

```diff
@@ -1,5 +1,14 @@
 # Migrating
 
+## Migrating from master to develop
+
+### Important changes
+* `CustomResetParameters` are now removed.
+* `reset()` on the Low-Level Python API no longer takes a `train_mode` argument. To modify the performance/speed of the engine, you must use an `EngineConfigurationChannel`
+* `reset()` on the Low-Level Python API no longer takes a `config` argument. `UnityEnvironment` no longer has a `reset_parameters` field. To modify float properties in the environment, you must use a `FloatPropertiesChannel`
+* The Academy no longer has a `Training Configuration` nor `Inference Configuration` field in the inspector. To modify the configuration form the Low-Level Python API, use an `EngineConfigurationChannel`. To modify it during training, use the new command line arguments `--width`, `--heigth`, `--quality-level`, `--time-scale` and `--target-frame-rate` in `mlagents-learn`.
```

Comment:
Spelling on `--height`

Also isn't this waaay too many options to do command line? I'd assume for most things you'd want to change all of them.

Reply:
For us, the defaults are pretty standard. Width and height are not really useful. Quality settings do not make a huge difference and can be left at default value. Target frame rate of -1 is default and should not change. The only one that really changes is **time scale**

Good catch on the typo

Reply:
Sounds good. So in training we'd just have them pass --time-scale=<whatevertheywant>. This is actually pretty nice, rather than baking it into the game.